### ISSUE TITLE
Background Music causes buffering in YouTube and popping when playing local music

### ISSUE THREAD

**Issue by samuelstanley0101** [kyleneideck/BackgroundMusic#783]
**Details**

I have Background Music playing audio through my headphones and have encountered several bugs. When playing a YouTube video, the video beings playing but immediately shows the buffering circle and, after about 5 seconds, stops playing. When my headphones are selected as the macOS output the video immediately continues playing without the buffering circle and there are no additional issues.

When playing any local audio through Background Music (tested with Apple Music and IINA) there is a slight popping sound every half second or so, which comes from seemingly random directions in stereo sound. This is most noticeable in quiet music but is still present in loud music. Sometimes the audio halts as if it were buffering. When my headphones are selected as the macOS output instead of Background Music these problems immediately stop.

I installed Background Music through `brew install --cask background-music` and, when I initially encountered this problem, removed it through `brew uninstall background-music` and then reinstalled it using the same command as before. Last night is when I reinstalled the program and I had no issues then but am having issues now.

**Versions**
 - Background Music: 0.4.3
 - macOS: Sequoia 15.2

**Comment by neilmenon** [comment]
I have this issue too. Did you figure out what's causing this?

**Comment by samuelstanley0101** [comment]
No, I ended up just uninstalling and dealing with manually adjusting volume for different apps.

**Comment by xianlihuang** [comment]
I have similar issue. When I open Background Music: 0.4.3. My bilibili is always buffering. It's really fast after I quit Background Music app. I use Macos 15.3

**Comment by xianlihuang** [comment]
> I have similar issue. When I open Background Music: 0.4.3. My bilibili is always buffering. It's really fast after I quit Background Music app. I use Macos 15.3

At the same time, there is not any sound when I connect a Bluetooth earphone. When I switch to the speaker, there is sound but breaking for a while.
